client/login/magic-login/request-login-email-form.jsx

```diff
@@ -185,7 +185,7 @@ class RequestLoginEmailForm extends Component {
 						{ tosComponent }
 						<div className="magic-login__form-action">
 							<FormButton primary disabled={ ! submitEnabled }>
-								{ submitButtonLabel || translate( 'Get Link' ) }
+								{ submitButtonLabel || translate( 'Send Link' ) }
```

Comment:
The same as above.

Reply:
@wellyshen - After discussing with @simison, we decided to just leave this new text in place for both places.